content/release-notes/1.16.0.md

```diff
@@ -0,0 +1,33 @@
+---
+date: 2020-06-01T08:00:00-07:00
+linktitle: "1.16.0"
+title: "1.16.0"
+kubernetes: "1.16, 1.17, and 1.18"
+---
+
+{{<features>}}
+* Support for HTTP_PROXY and NO_PROXY environment variables.
+* CronJobs now support private images.
+* Gitops cluster tracers
+* Scheduled update checks
+* Support for custom redactors
+{{</features>}}
+
+{{<changes>}}
+* Velero storage credentials will now be validated on save.
+* Updated troubleshoot dependency to v0.9.36
+* Helm 3 support
+* ConfigItem's `when` can now be a boolean
+* Namespaces will be applied before the rest of the application YAML.
+* Velero and Restic information is included in support bundles
+{{</changes>}}
+
+{{<fixes>}}
+* Fixed a bug that could cause `LocalImageName` and `LocalRegistryImagePullSecret` to panic.
+* Snapshots page will show a message when connection to the API is lost.
+* Fixed a bug that would cause orphaned pods to be left on the system when jobs are deleted.
+* Fixed a bug that could cause a mismatch between TLS cert and key when using `TLSCert` and `TLSKey` template functions
+* Added scrolling on airgap upload progress page.
+* Fixed a bug that allowed installing application with invalid YAML.
```

Comment:
What happened when you installed with invalid YAML?

Reply:
How about `Warnings will now be visible in the browser admin console when installing an application with invalid YAML`?

Reply:
We should remove this. There is no change for this release